Sources/Support/DebugUI/DebugViewSheetPresentation.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  DebugViewSheetPresentation.swift
+//
+//  Created by Nacho Soto on 5/30/23.
+
+#if DEBUG && os(iOS) && swift(>=5.8)
+
+import SwiftUI
+
+@available(iOS 16.0, *)
+extension View {
+
+    @ViewBuilder
+    func bottomSheet(
+        presentationDetents: Set<PresentationDetent>,
+        isPresented: Binding<Bool>,
+        largestUndimmedIdentifier: PresentationDetent = .large,
+        cornerRadius: CGFloat,
+        interactiveDismissDisabled: Bool = false,
+        @ViewBuilder content: @escaping () -> some View
+    ) -> some View {
+        self
+            .sheet(isPresented: isPresented) {
```

Comment:
I'm not familiar with iOS UI development 😅 . So just to check, if `isPresented` is false, is the view created? I just have some concerns of loading the debug view model triggering some extra operations in the SDK... I don't think it should be a big issue right now though.

Reply:
Good point let me check

Reply:
Confirmed :) it doesn't load until it's visible.